modules/virt-using-virt-must-gather.adoc

```diff
@@ -24,5 +24,5 @@ The default data collection includes information about the following resources:
 [source,terminal,subs="attributes+"]
 ----
 $ oc adm must-gather --image-stream=openshift/must-gather \
-  --image=registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel9:v{HCOVersion}
+  --image=registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel9:v{HCOVersion} – /usr/bin/gather 
```

Comment:
This will be :

~~~
 oc adm must-gather --image-stream=openshift/must-gather \
  --image=registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel9:v{HCOVersion} /usr/bin/gather
  ~~~